src/test/java/org/apache/commons/net/util/Base64Test.java

```diff
@@ -208,9 +241,10 @@ public void testEncodeObject() {
     }
 
     @Test
-    @Ignore
     public void testEncodeToString() {
-        fail("Not yet implemented");
+        final Base64 base64 = new Base64();
+        final byte[] bytesToEncode = new byte[] {'l', 'i', 'g', 'h', 't', ' ', 'w', 'o', 'r'};
```

Comment:
Random question here; was the example intended to be "light work" encoded? :slightly_smiling_face: Asking as it was used in the other tests, but doesn't really matter as this test is working fine :+1:

Reply:
nope, just a random example from [Base64 - Output Padding](https://en.wikipedia.org/wiki/Base64#Output_padding) page in wikipedia 😀

Reply:
> Random question here; was the example intended to be "light work" encoded? 🙂 Asking as it was used in the other tests, but doesn't really matter as this test is working fine 👍

In the future, we could split up test method per test source, like `testWikipediaExamplesAbc`, `testRfc123FeatureX`, and so on.